src/static/js/ethers_helper.js

```diff
@@ -1190,11 +1190,13 @@ async function getJar(app, jar, address, stakingAddress) {
 
 async function getCurveToken(app, curve, address, stakingAddress, minterAddress) {
   const minter = new ethcall.Contract(minterAddress, MINTER_ABI)
-  const [virtualPrice, coin0] = await app.ethcallProvider.all([minter.get_virtual_price(), minter.coins(0)]);
-  const token = await getToken(app, coin0, address);
-  const calls = [curve.decimals(), curve.balanceOf(stakingAddress), curve.balanceOf(app.YOUR_ADDRESS),
-    curve.name(), curve.symbol(), curve.totalSupply()];
-  const [decimals, staked, unstaked, name, symbol, totalSupply] = await app.ethcallProvider.all(calls);
+  const [virtualPrice, coin0] = await app.ethcallProvider.all([minter.get_virtual_price(), minter.coins(0)]);  
+  
+  const lpToken = new ethcall.Contract(curve.address, CURVE_ABI);
```

Comment:
The `curve` var here is the same thing as `lpToken`, not sure what the difference in these changes is